### PR TITLE
my attempted solution codes written for exercise 2

### PR DESCRIPTION
**Pull request by kingsedem**
These are my attempted solutions for Exercise 2(https://expansion.serenity-dojo.com/courses/take/serenity-dojo-java-for-testers/texts/37287591-exercise-java-data-types) 

Kindly review.

### CODE REVIEW

File: src/test/java/com/serenitydojo/WhenWorkingWithFloatingPointNumbers.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import java.io.FilterOutputStream;
+
+public class WhenWorkingWithFloatingPointNumbers {
+    @Test
+// test to convert a number from celsius to fahrenheit
+    public void convertCelsiusToFahrenheit(){
+
+        double celsius = 50.0;
+        double fahrenheit = 0.0;
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWorkingWithFloatingPointNumbers.java]: initializing with 0 doesn't really do anything here. cleaner code would be to directly assign the value we need like  
```suggestion
        double fahrenheit = (celsius * 1.8) + 32;
```

File: src/test/java/com/serenitydojo/WhenWorkingWithStrings.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.serenitydojo;
+
+
+import org.junit.Test;
+
+public class WhenWorkingWithStrings {
+
+    //Exercise 1 - Converting to lower case
+@Test
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWorkingWithStrings.java]: line intendation seems inconsistent. you want something that's automatically fixing that.  with intellij you can configure that java files are automatically formatted on saving https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat-on-save

File: src/test/java/com/serenitydojo/WhenWorkingWithStrings.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    public void ConvertingToUpperCase(){
+
+    String myFavoriteColor = "Green";
+    System.out.println(myFavoriteColor.toUpperCase());
+}
+
+// Exercise 3 - Trim extra space
+@Test
+    public void TrimExtraSpace(){
+    String myCountry = "Nigeria";
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWorkingWithStrings.java]: that doesnt really show trimming. adding some space around the word would show it better
```suggestion
    String myCountry = "   Nigeria    ";
```

File: src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.junit.Test;
+
+
+public class WhenWorkingWithWholeNumbers {
+
+    @Test
+    public void addNumbersTogether(){
+        int initialYear =  1985;
+        int targetYear  = 0;
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java]: same as https://github.com/serenity-dojo/exercise-1-kingsedem/pull/1/files#r1069047911

File: src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+        int timeJump = 30;
+        targetYear =  initialYear + timeJump;
+
+        System.out.println(targetYear);
+        //assertThat(targetYear, equalTo(2015));
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java]:
```suggestion
```
if it is not needed, remove it I'd say

**Review by globalworming — APPROVED** [review]

File: src/test/java/com/serenitydojo/WhenWorkingWithFloatingPointNumbers.java (hunk 1 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWorkingWithFloatingPointNumbers.java]: Thank you for the feedback

File: src/test/java/com/serenitydojo/WhenWorkingWithStrings.java (hunk 2 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWorkingWithStrings.java]: Done.

File: src/test/java/com/serenitydojo/WhenWorkingWithStrings.java (hunk 3 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWorkingWithStrings.java]: Done

File: src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java (hunk 4 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java]: Done

File: src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java (hunk 5 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java]: Done

File: src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java (hunk 4 above)
Comment (globalworming) [reply src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java]: not pushed yet?

**Pull request merged**